lib/model/folder_sendrecv.go

```diff
@@ -743,7 +743,7 @@ func (f *sendReceiveFolder) handleSymlink(file protocol.FileInfo, snap *db.Snaps
 		return
 	}
 
-	if f.handleSymlinkCheckExisting(file, snap, scanChan); err != nil {
+	if err = f.handleSymlinkCheckExisting(file, snap, scanChan); err != nil {
```

Comment:
```suggestion
	if err := f.handleSymlinkCheckExisting(file, snap, scanChan); err != nil {
```

is the more common spelling, you variant is equivalent of course but makes me starting looking for possible side effects of assigning to err in the outer scope

Reply:
I was just doing what the comment told me :)

Reply:
Oh so there was a side effect, intended. I'm not sure that solution is beautiful exactly, but it's not your fault at least. :)